liota/dccs/iotcc.py

```diff
@@ -346,7 +391,7 @@ def _create_iotcc_json(self):
             log.error('Could not open {0} file '.format(iotcc_path) + err)
         return iotcc_path
 
-    def store_reg_entity_details(self, entity_type, entity_name, reg_entity_id, entity_local_uuid):
+    def _store_reg_entity_details(self, entity_type, entity_name, reg_entity_id, entity_local_uuid):
```

Comment:
Though these are private methods, documentation would be useful from developer perspective